src/brainbrowser/surface-viewer/modules/rendering.js

```diff
@@ -623,6 +631,99 @@ BrainBrowser.SurfaceViewer.modules.rendering = function(viewer) {
     return vertex_data;
   };
 
+
+  /**
+  * @doc function
+  * @name viewer.rendering:modelCentric
+  * @param {boolean} if true, recenter all userData shape on origin. Otherwise
+  * return to the original userData.
+  *
+  * @description
+  * Use to recenter data when userData input is shifted in space.
+  *
+  *
+  * ```js
+  * viewer.modelCentric(true);
+  * ```
+  */
+  viewer.modelCentric = function(model_centric) {
+    if (model_centric === undefined) {model_centric = false;}
```

Comment:
Ok I will fix it, and I'm kind of agree with you. 
In fact I like used `return if something` but it is not supported in JS.